crates/uv-cli/src/lib.rs

```diff
@@ -2711,23 +2712,27 @@ pub struct RunArgs {
     #[arg(long, conflicts_with_all = ["only_group", "only_dev"])]
     pub group: Vec<GroupName>,
 
-    /// Exclude dependencies from the specified dependency group.
+    /// Disable the specified dependency group.
```

Comment:
Notably this specific wording gave me a big misconception early on. I thought the exclusion was at the level of the dependencies references, and not just the names of the groups themselves.